src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -936,11 +935,13 @@ public AudioManager getAudioManager()
         }
 
         AudioChannel channel = getSelfMember().getVoiceState().getChannel();
-        StageInstance instance = channel instanceof StageChannel ? ((StageChannel) channel).getStageInstance() : null;
-        if (instance == null)
-            return new GatewayTask<>(CompletableFuture.completedFuture(null), () -> {});
-        CompletableFuture<Void> future = instance.cancelRequestToSpeak().submit();
-        return new GatewayTask<>(future, () -> future.cancel(false));
+        if (channel instanceof StageChannel)
+        {
+            CompletableFuture<Void> future = ((StageChannel) channel).cancelRequestToSpeak().submit();
+            return new GatewayTask<>(future, () -> future.cancel(false));
```

Comment:
Why are these `GatewayTasks` when the actual request is a `RestAction`?

Reply:
Is it just so that `requestToSpeak` and `cancelRequestToSpeak` have a mirrored return type?

Reply:
Symmetry and my hopes that this will in the future be a request you can do through the gateway